apps/condo/domains/common/components/GraphQlSearchInput.tsx

```diff
@@ -65,6 +65,11 @@ export const GraphQlSearchInput: React.FC<ISearchInputProps> = (props) => {
     function handleSelect (value, option) {
         setSelected(option.children)
 
+        if (props.mode === 'multiple') {
+            setValue('')
+            handleSearch('')
```

Comment:
It triggers extra query on select item.
```suggestion
```
Can `autoClearSearchValue` property of `Select` be used in your case?
https://ant.design/components/select/
I've tried locally, it seems to be working.